pod-security/restricted/require-non-root-groups/require-non-root-groups.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   annotations:
     policies.kyverno.io/category: Pod Security Standards (Restricted)
     policies.kyverno.io/severity: medium
+    policies.kyverno.io/minversion: 1.6.0
```

Comment:
Hi @smnmtzgr - the minimum version should be 1.3.6.

Reply:
sorry fixed...